src/library_wasi.js

```diff
@@ -90,15 +90,16 @@ var WasiLibrary = {
   args_sizes_get__nothrow: true,
   args_sizes_get__sig: 'iii',
   args_sizes_get: function(pargc, pargv_buf_size) {
+    {{{ from64(['pargc', 'pargv_buf_size']) }}};
 #if MAIN_READS_PARAMS
-    {{{ makeSetValue('pargc', 0, 'mainArgs.length', 'i32') }}};
+    {{{ makeSetValue('pargc', 0, 'mainArgs.length', `${SIZE_TYPE}`) }}};
```

Comment:
I think you can just using `SIZE_TYPE` rather than wrapping it backticks+dollar+curlies.    Its already a string, isn't it?   Here and everywhere else in this file.